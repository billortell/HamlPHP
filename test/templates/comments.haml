!!! 5
%html
  %body
    %p
      / one line html comment
    %p
      /
        multiline
        html
        comment
        %p jee
    %p
      -# one line comment
    %p
      -#
        multi
        line
        comment
    %p
      /[if IE]
        this is visible only for IE