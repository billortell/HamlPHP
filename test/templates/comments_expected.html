<!DOCTYPE html>
<html>
  <body>
    <p>
      <!-- one line html comment -->
    </p>
    <p>
      <!--
        multiline
        html
        comment
        <p>jee</p>
      -->
    </p>
    <p>

    </p>
    <p>

    </p>
  </body>
</html>
